app/src/main/java/com/ecosystem/kin/app/App.java.orig

```diff
@@ -0,0 +1,133 @@
+package com.ecosystem.kin.app;
```

Comment:
delete this file - a leftover from merge?

Reply:
thanks 👍